include/tvm/ir/memory_pools.h

```diff
@@ -130,15 +129,159 @@ static const int kUnknownReadBandwidth = -1;
 static const int kUnknownWriteBandwidth = -1;
 
 class PoolInfo : public ObjectRef {
```

Comment:
could you add/update docstrings for each of these classes?

Reply:
Added